glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -488,17 +488,47 @@ open class GleanInternalAPI internal constructor () {
     }
 
     /**
-     * Should be called from all users of the Glean testing API.
+     * TEST ONLY FUNCTION.
+     * This is called by the GleanTestRule, to enable test mode.
      *
      * This makes all asynchronous work synchronous so we can test the results of the
      * API synchronously.
      */
-    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
-    fun enableTestingMode() {
+    internal fun enableTestingMode() {
         @Suppress("EXPERIMENTAL_API_USAGE")
         Dispatchers.API.setTestingMode(enabled = true)
     }
 
+    /**
+     * TEST ONLY FUNCTION.
+     * Resets the Glean state and trigger init again.
+     *
+     * @param context the application context to init Glean with
+     * @param config the [Configuration] to init Glean with
+     * @param clearStores if true, clear the contents of all stores
+     */
+    internal fun resetGlean(
+        context: Context,
+        config: Configuration,
+        clearStores: Boolean
+    ) {
+        Glean.enableTestingMode()
+
+        /*if (clearStores) {
```

Comment:
Is this commented out because we don't have it in glean-core (yet)?
If so I would expect at least a `TODO: ...` to at least find it again (or even a bug to bring it back [or check if we _need_ to bring it back])

If it's clear it's not coming back we can just removed commented out code.

Reply:
This code was simply moved from `TestUtils.ks`. I filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1570247) for it.